messaging/methods/media/uploadMedia.md

```diff
@@ -50,7 +50,9 @@ var a = b;
 {% sample lang="ruby" %}
 
 ```ruby
-# Ruby Example
+f = File.open("some file", "r")
+file_content = f.read
+messaging_client.upload_media("12345", "mediaId", file_content.length.to_s, file_content, :content_type => "application/octet-stream", :cache_control => "no-cache")
```

Comment:
Does this return anything? 

Do I get a new URL ?

Reply:
No return according to the OAS